### PR TITLE
adding two new options, custom filename and filename in output dir

### LINKED ISSUE #85: Feature Request - Markdown filename (without .md) as part of output path
@bingryan any chance this can be extended with an option to include the markdown filename, without the extension as part of the output path?

### PR DESCRIPTION
**Pull request by shindakun**
closes #85 

This PR will close #85, not sure if my code is up to snuff but it seems to work as expected. Putting up this PR just in case someone else wants a bit more control over the output like I did.

### CODE REVIEW

**Comment by bingryan** [comment]
@shindakun thx for you pr, fix error plz 👍

**Comment by bingryan** [comment]
@shindakun hi shindakun, can you merge latest master as first ?

**Comment by shindakun** [comment]
> @shindakun hi shindakun, can you merge latest master as first ?

It should be as of the last commit.

**Review by bingryan — CHANGES_REQUESTED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -72,10 +72,16 @@ export default class MarkdownExportPlugin extends Plugin {
...
+		let dir = ""
+		if (this.settings.includeFileName == true) {
+			dir = file.name.replace(".md", "")
+		}
+
 		// try create attachment directory
 		await tryCreateFolder(
 			this,
-			path.join(this.settings.output, this.settings.attachment),
+			path.join(this.settings.output, dir, this.settings.attachment),
```
[the remark this answers is not in the document]
Comment (shindakun) [reply src/main.ts:84]: So the files save into the correct directory if you have the option enabled

**Review by bingryan — APPROVED** [review]

**Pull request merged**